docs/Chipyard-Basics/Chipyard-Components.rst

```diff
@@ -24,6 +24,10 @@ Processor Cores
   An in-order RISC-V core written in System Verilog. Previously called Ariane.
   See :ref:`Generators/CVA6:CVA6 Core` for more information.
 
+**Ibex Core**
+  An in-order RISC-V core writeen in System Verilog.
```

Comment:
```suggestion
  An in-order RISC-V core written in System Verilog.
```

Reply:
I would expand on this a bit and say its a 32b core only to distinguish it between all the other cores which are potentially 64 or 32b.